seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/format/HBaseDeserializationFormat.java

```diff
@@ -60,16 +60,15 @@ private Object deserializeValue(SeaTunnelDataType<?> typeInfo, byte[] cell) {
             case TINYINT:
             case SMALLINT:
             case INT:
-                return Bytes.toInt(cell);
+                return Integer.valueOf(Bytes.toString(cell));
```

Comment:
**Issue**: Converting byte arrays to strings and then to integers could lead to performance degradation and potential NumberFormatException if the byte array content is not a valid string representation of an integer. <br> **Fix**: It's recommended to revert to the direct conversion from byte array to integer to avoid unnecessary performance overhead and potential errors. <br> **Code Suggestion**: 
 ```
 -                return Integer.valueOf(Bytes.toString(cell));
 +                return Bytes.toInt(cell);
 ```